tools/maint/gen_sig_info.py

```diff
@@ -166,12 +166,20 @@
 
 
 def ignore_symbol(s, cxx):
+  # We need to ignore certain symbols here. Specifically any symbol that doesn't
+  # appear in C/C++ header file.
```

Comment:
```suggestion
  # appear in a C/C++ header file (such symbols cannot be called by JS).
```
Or is there another reason for why we need to ignore these?

Reply:
No, its more like "Such symbols cannot be part of the auto-sig-generation process at all since we depend on import these functions with their header-declared signature."

Reply:
Its just the way gen_sig_info works:  It generated a bunch of references to header-declared symbols.  If they symbol are not declared the file cannot be compiled.

Reply:
Oh, right, thanks.